src/org/toml/lang/icons/TomlIcons.kt

```diff
@@ -1,12 +1,13 @@
 package org.toml.lang.icons
 
+import com.intellij.icons.AllIcons
 import com.intellij.openapi.util.IconLoader
 import javax.swing.Icon
 
 
 public object TomlIcons {
-    public val NORMAL: Icon = IconLoader.getIcon("/org/toml/icons/cargo16.png")
-    public val BIG: Icon = IconLoader.getIcon("/org/toml/icons/cargo32.png")
+    public val FILE = AllIcons.FileTypes.Text;
```

Comment:
Nope, it has not at the moment :(

The only thing you can do is to set semicolon color to bold blood red :)